DeprecationWarning generated under latest pytest

As of the release of pytest 5.4.0 earlier today, attempting to run pytest with pytest-flake8 installed and with `[pytest]filterwarnings` set to `error` causes the run to fail immediately with:

```
.tox/py36/lib/python3.6/site-packages/pluggy/hooks.py:286: in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
.tox/py36/lib/python3.6/site-packages/pluggy/manager.py:93: in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
.tox/py36/lib/python3.6/site-packages/pluggy/manager.py:87: in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
.tox/py36/lib/python3.6/site-packages/pytest_flake8.py:72: in pytest_collect_file   
    statistics=config._flake8statistics)
.tox/py36/lib/python3.6/site-packages/_pytest/nodes.py:83: in __call__
    warnings.warn(NODE_USE_FROM_PARENT.format(name=self.__name__), stacklevel=2)
E   pytest.PytestDeprecationWarning: direct construction of Flake8Item has been deprecated, please use Flake8Item.from_parent
```

This is due to the deprecation of direct constructors for `Nodes`; see item 5975 under "Deprecations" in [the release notes for pytest 5.4.0](https://docs.pytest.org/en/latest/changelog.html#pytest-5-4-0-2020-03-12).  pytest-flake8 thus needs to be updated; according to the release notes:

> Subclasses are expected to use `super().from_parent` if they intend to expand the creation of `Nodes`.

--- a/pytest_flake8.py
+++ b/pytest_flake8.py
@@ -62,14 +62,23 @@ def pytest_collect_file(path, parent):
     if config.option.flake8 and path.ext in config._flake8exts:
         flake8ignore = config._flake8ignore(path)
         if flake8ignore is not None:
-            return Flake8Item(
-                path,
-                parent,
-                flake8ignore=flake8ignore,
-                maxlength=config._flake8maxlen,
-                maxcomplexity=config._flake8maxcomplexity,
-                showshource=config._flake8showshource,
-                statistics=config._flake8statistics)
+            if hasattr(Flake8Item, "from_parent"):
+                item = Flake8Item.from_parent(parent, fspath=path)
+                item.flake8ignore = flake8ignore
+                item.maxlength = config._flake8maxlen
+                item.maxcomplexity = config._flake8maxcomplexity
+                item.showshource = config._flake8showshource
+                item.statistics = config._flake8statistics
+                return item
+            else:
+                return Flake8Item(
+                    path,
+                    parent,
+                    flake8ignore=flake8ignore,
+                    maxlength=config._flake8maxlen,
+                    maxcomplexity=config._flake8maxcomplexity,
+                    showshource=config._flake8showshource,
+                    statistics=config._flake8statistics)
 
 
 def pytest_unconfigure(config):

